From dca906e02f72342527746ff33a827196a3f730ad Mon Sep 17 00:00:00 2001
From: Pierre-Yves MORDRET <pierre-yves.mordret@foss.st.com>
Date: Tue, 25 Jun 2024 13:59:56 +0200
Subject: [PATCH] arm64: dts: st: Change axi to bus for GPU

To comply with Vivante.yaml bindings change axi clock to bus clock

Change-Id: Ifcb22c611183cf082152f54369e7f39ceebe6953
Signed-off-by: Pierre-Yves MORDRET <pierre-yves.mordret@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/389525
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/boot/dts/st/stm32mp255.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp255.dtsi b/arch/arm64/boot/dts/st/stm32mp255.dtsi
index 9e6074d4f23b..861431eafe3b 100644
--- a/arch/arm64/boot/dts/st/stm32mp255.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp255.dtsi
@@ -63,7 +63,7 @@ gpu: gpu@48280000 {
 		reg = <0x48280000 0x800>;
 		interrupts = <GIC_SPI 183 IRQ_TYPE_LEVEL_HIGH>;
 		resets = <&rcc GPU_R>;
-		clock-names = "axi", "core";
+		clock-names = "bus", "core";
 		clocks = <&rcc CK_BUS_GPU>, <&rcc CK_KER_GPU>;
 		power-domains = <&scmi_devpd PD_SCMI_GPU>;
 		access-controllers = <&rifsc 79>;
-- 
2.39.5

